src/vscode-bicep/src/language/client.ts

```diff
@@ -14,6 +14,7 @@ import { ErrorAction, Message, CloseAction } from "vscode-languageclient/node";
 
 const dotnetRuntimeVersion = "3.1";
 const packagedServerPath = "bicepLanguageServer/Bicep.LangServer.dll";
+const extensionId = "ms-azuretools.vscode-bicep";
```

Comment:
No way to get the current extension's ID?

Reply:
Not that I'm aware of. This is the pattern I saw in other consumers of the dotnet runtime extension.

Reply:
Sounds good.